census/ui_app/HelperFields.jsx

```diff
@@ -66,6 +66,8 @@ const SubmitActions = props => {
       <div className="comments"></div>
     </div>
     </div>);
+  } else if (props.isReview && !props.canReview) {
```

Comment:
Second condition is not really needed in this case, since it's not getting here if `props.canReview` is falsy.